Taskfile.yml

```diff
@@ -75,28 +80,61 @@ tasks:
       - cmd: echo "submodules ready!"
         silent: true
 
-  has-latest-opslevel-go:
-    desc: Check if latest release of opslevel-go in go.mod
+  has-latest-opslevel-dependencies:
+    desc: Check if opslevel dependencies are up to date in go.mod
     dir: "{{.SRC_DIR}}"
     silent: true
     vars:
       GO_WORK: "go.work"
       TMP_GO_WORK: "tmp_go.work"
-      LATEST_OPSLEVEL_GO_VERSION:
-        sh: go list -u -m -versions github.com/opslevel/opslevel-go/v2024 | awk '{print $NF}'
+    cmds:
+      - task: tmp-ignore-go-work-file
+      - defer: { task: undo-ignore-go-work-files }
+      - for: [ "{{.OPSLEVEL_GO_PKG}}", "{{.OPSLEVEL_JQ_PARSER_PKG}}", "{{.OPSLEVEL_K8S_CONTROLLER_PKG}}" ]
+        task: has-latest-opslevel-dep
+        vars:
+          OPSLEVEL_PKG: "{{.ITEM}}"
+
+  # internal (not directly called) tasks
+
+  tmp-ignore-go-work-file:
+    dir: "{{.SRC_DIR}}"
+    internal: true
+    silent: true
+    vars:
+      GO_WORK: "go.work"
+      TMP_GO_WORK: "tmp_go.work"
     cmds:
       - if [[ -f {{.GO_WORK}} ]]; then mv {{.GO_WORK}} {{.TMP_GO_WORK}}; fi
-      - defer: if [[ -f {{.TMP_GO_WORK}} ]]; then mv {{.TMP_GO_WORK}} {{.GO_WORK}}; fi
+
+  undo-ignore-go-work-files:
+    dir: "{{.SRC_DIR}}"
+    internal: true
+    silent: true
+    vars:
+      GO_WORK: "go.work"
```

Comment:
I noticed `GO_WORK` and `TMP_GO_WORK` are always the exact same string value - if that's the case can we just get rid of the variables and use the literals?

Reply:
Yeah, this used to make sense but doesn't anymore. Update coming